src/setup.ts

```diff
@@ -6,13 +6,13 @@ const setup = () => {
         for (let y = 0; y < board[x].length; y++) {
             const square = document.createElement('div');
             square.id = `${x},${y}`;
-            square.innerHTML = board[x][y] ? board[x][y].display : '';
+            square.innerHTML = board[x][y] ? board[x][y]!.display : '';
```

Comment:
To jest tutaj na pewno nie potrzebne, jeżeli nie będzie czegos w board to z automatu idzie na false więc usuńmy to